sru-templates/manual/azure-sru

```diff
@@ -15,11 +17,42 @@ RESOURCE_GROUP="<specify your resource group>"
 BOOT_DIAG="<specify your boot diagnostics group>"
 SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
+
+IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+
+if [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; then
+	echo "Account is not yet registered for AllowIPv6VirtualNetwork or AllowIPv6CAOnStandardLB. Can take 30 mins. Activate feature now (y/n)?"
```

Comment:
This indentation is a tab, not spaces. (This is so much easier for you to fix with your editor that I'm not going to suggest a change here for each one. :-P)

Reply:
Yes, please no tabs